### ISSUE TITLE
Accessibility - remove empty headings 

### ISSUE THREAD

**Issue by id2win** [GSA/idmanagement.gov#604]
Accessibility issue rating - A

Issue - pages contain empty headings; 17 occurrences on this page. 

Details - https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?pageId=38246460841&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r64/failed/UDBnb0hnVTg=;ZTQ0ZGNiYzY=

Fix - remove empty headings.
